gym/envs/box2d/car_racing.py

```diff
@@ -517,10 +517,11 @@ def reset(
             self.render()
         return self.step(None)[0], {}
 
-    def step(self, action: Union[np.ndarray, int]):
+    def step(self, action: Optional[Union[np.ndarray, int]]):
         assert self.car is not None
         if action is not None:
             if self.continuous:
+                assert isinstance(action, np.ndarray)
```

Comment:
Sorry on the discussion on `np.clip` I was talking about Car Racing not Lunar Lander, my mistake

Reply:
There is no call to `np.clip` in CarRacing. So passing a tensor would cause an error when the environment internals try to use the elements of the tensor. We can alleviate this by additionally calling `action = np.array(action)` which ensures tensor conversion to numpy arrays. 
Hence, as it is, the environment would not work with tensors, even without the assert that ensures actions are numpy arrays.

Reply:
I would remove it for now